presto-main/src/test/java/com/facebook/presto/operator/spiller/BenchmarkBinaryFileSpiller.java

```diff
@@ -142,6 +142,7 @@ public void tearDown()
                     DOUBLE.writeDouble(pageBuilder.getBlockBuilder(4), lineItem.getExtendedPrice());
                 }
                 pages.add(pageBuilder.build());
+                pageBuilder.reset();
```

Comment:
👍 , you could also update the commit message (but mind the guidelines https://chris.beams.io/posts/git-commit/). e.g

```
Fix PageBuilder use in benchmark binary file spiller

`PageBuilder` needs to be `reset()` between building pages.
```